src/main/java/juja/microservices/keepers/dao/KeepersRepository.java

```diff
@@ -42,34 +44,44 @@
     public String save(KeeperRequest keeperRequest) {
         Date startDate = Date.from(LocalDateTime.now().atZone(ZoneId.systemDefault()).toInstant());
         Keeper keeper = new Keeper(keeperRequest.getFrom(),
```

Comment:
I think this logic of creating Keeper entity from KeeperRequest should be in the service level. Then in the repository we can use only one method `save` to save new keeper or update existing one. This way the method `inactive` will be redundant.